charts/media-common/Chart.yaml

```diff
@@ -2,10 +2,16 @@ apiVersion: v2
 name: media-common
 description: Common dependancy chart for media ecosystem containers
 type: application
-version: 1.0.1
+version: 1.1.0
 keywords:
   - media-common
 home: https://github.com/k8s-at-home/charts/tree/master/charts/media-common
 maintainers:
   - name: DirtyCajunRice
     email: nick@cajun.pro
+dependencies:
+  - name: media-common-openvpn
+    repository: https://k8s-at-home.com/charts/
+    version: 1.0.0
```

Comment:
Lets preemptively set this to  `^1.0.0` so that we can accomodate for minor updates and micro updates of the library chart